src/pal/inc/pal.h

```diff
@@ -3649,6 +3649,19 @@ LoadLibraryExW(
         IN /*Reserved*/ HANDLE hFile,
         IN DWORD dwFlags);
 
+PALIMPORT
+HMODULE
+PALAPI
+LoadLibraryDirectW(
```

Comment:
Also noticed there is already a PAL_RegisterLibraryW, renamed the new function to PAL_RegisterLibraryDirect